app/experimenter/nimbus-ui/src/components/PageEditAudience/index.test.tsx

```diff
@@ -55,12 +73,36 @@ describe("PageEditAudience", () => {
     render(<Subject mocks={[mock, mutationMock]} />);
     await screen.findByTestId("PageEditAudience");
     fireEvent.click(screen.getByTestId("next"));
+    expect(experiment.status).toEqual(NimbusExperimentStatusEnum.DRAFT);
     await waitFor(() => {
       expect(mockSubmit).toHaveBeenCalled();
       expect(navigate).toHaveBeenCalledWith("../");
     });
   });
 
+  it("handles form next button for live updates", async () => {
+    render(<Subject mocks={[mockRollout, rolloutMutationMock]} />);
+    await screen.findByTestId("PageEditAudience");
+    fireEvent.click(screen.getByTestId("next"));
+    expect(rollout.status).toEqual(NimbusExperimentStatusEnum.LIVE);
+    await waitFor(() => {
+      expect(mockSubmit).toHaveBeenCalled();
+      expect(navigate).toHaveBeenCalledWith("summary");
+    });
+  });
+
+  it("handles save button for live updates", async () => {
+    const { mockRollout, rollout } = mockLiveRolloutQuery("demo-slug", {
+      status: NimbusExperimentStatusEnum.LIVE,
+      publishStatus: NimbusExperimentPublishStatusEnum.IDLE,
+      isRollout: true,
+    });
+    render(<Subject mocks={[mockRollout, rolloutMutationMock]} />);
+    await screen.findByTestId("PageEditAudience");
+    fireEvent.click(screen.getByTestId("submit"));
+    await waitFor(() => expect(mockSubmit).toHaveBeenCalled());
```

Comment:
So what exactly is this case testing?  Is there specific logic that says that this save won't occur if it's live but not a rollout?

Reply:
That's what I mean, the logic is preventing navigation to this page, but not preventing saving, so without an explicit logic check to control when save is available, I'm not sure this test is actually testing anything?  Not a big deal though.